### ISSUE TITLE
[BUG]: Humble build fails due to missing dependency

### ISSUE THREAD

**Issue by evan-palmer** [Robotic-Decision-Making-Lab/blue#25]
### Issue Description

The Humble Docker build fails when attempting to install the rosdeps for `ros_gz`.

### Steps to Reproduce

The failed build can be observed [here](https://github.com/evan-palmer/blue/actions/runs/5315681667/jobs/9624346380)

### Expected Behavior

The build is able to retrieve all rosdeps successfully.

### Error Message

```bash
ros_gz_bridge: Cannot locate rosdep definition for [rosidl_pycommon]
```

**Comment by evan-palmer** [comment]
I removed support for the `source` stage here #52. Going to close this for now until I need to integrate that stage back in.

**Comment by evan-palmer** [comment]
Alright this bug has re-emerged now that I am adding stages back in. I'll resolve this once I finished the fixes for #120
